pkg/operator/loglevel/logging_controller.go

```diff
@@ -28,24 +35,37 @@ func (c LogLevelController) sync(ctx context.Context, syncCtx factory.SyncContex
 		return err
 	}
 
-	currentLogLevel := CurrentLogLevel()
+	currentLogLevel, isUnknown := c.currentLogLevelFn()
 	desiredLogLevel := detailedSpec.OperatorLogLevel
 
+	// if operator operatorSpec OperatorLogLevel is empty, default to "Normal"
+	// TODO: This should be probably done by defaulting the CR field?
```

Comment:
doesn't this apply to startup as well when loglevel is not set?

Reply:
yes.